src/explore-education-statistics-admin/src/pages/release/content/components/EditableKeyStatTextForm.tsx

```diff
@@ -21,19 +23,19 @@ export interface KeyStatTextFormValues {
 }
 
 interface EditableKeyStatTextFormProps {
-  keyStat: KeyStatisticText;
-  isReordering?: boolean;
+  keyStat?: KeyStatisticText;
   onSubmit: (values: KeyStatTextFormValues) => void;
   onCancel: () => void;
-  testId?: string;
+  isReordering?: boolean;
+  testId: string;
```

Comment:
Bit of a nitpick - why have we moved these? Although we're a bit inconsistent, we typically place `on*` event handler props last. Depending on the case, we may also alphabetise the rest of the props as well.

Reply:
Honestly can't remember - I've moved them back.